modules/oe_theme_content_publication/src/Plugin/ExtraField/Display/PublicationDate.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_publication\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Displays publication date and last update date fields.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_theme_content_publication_date",
+ *   label = @Translation("Publication date"),
+ *   bundles = {
+ *     "node.oe_publication",
+ *   },
+ *   visible = true
+ * )
+ */
+class PublicationDate extends ExtraFieldDisplayFormattedBase implements ContainerFactoryPluginInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * PublicationDate constructor.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Datetime\DateFormatterInterface $date_formatter
+   *   Date formatter.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, DateFormatterInterface $date_formatter) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->dateFormatter = $date_formatter;
```

Comment:
This is not declared as a property of the class.

Reply:
Fixed